blacklisted-tokens.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+      "name": "hausemater topg", 
```

Comment:
@RitikJ22 Use `trim().toLowerCase().replace(/[^\w\d-]/g, '-')` for names. It doesn't follow the naming convention